bench-tps/src/main.rs

```diff
@@ -146,7 +146,9 @@ fn main() {
         target /= MAX_SPENDS_PER_TX;
     }
     let gen_keypairs = rnd.gen_n_keypairs(total_keys as u64);
-    let barrier_id = rnd.gen_n_keypairs(1).pop().unwrap();
+    let mut barrier_ids = rnd.gen_n_keypairs(2);
```

Comment:
While we’re here, an id is a Pubkey not a Keypair.  You could rename barrier_id to barrier_keypair, and barrier_dest_id to barrier_dest_keypair to resolve

Reply:
Barrier_dest_id is a pubkey. I'll update the barrier_id to barrier_keypair though

Reply:
barrier_source_id too please